www/gatsby-config.js

```diff
@@ -53,12 +53,8 @@ if (process.env.AIRTABLE_API_KEY) {
   })
 }
 
-// true if `env.LOCALES` has a defined list of languages
-if (langCodes.length > 0) {
-  const naughtyFiles = [
-    `docs/docs/graphql-api.md`,
-    `docs/docs/data-fetching.md`,
-  ]
+if (i18nEnabled) {
+  const naughtyFiles = [`docs/docs/data-fetching.md`]
```

Comment:
`naughtyFiles` should probably be named something else. These are the pages that use one-off MDX components, yeah? Even I wouldn't have known that if I hadn't _just_ looked at issue #20795.